core/src/mindustry/game/Rules.java

```diff
@@ -90,6 +90,12 @@
     public boolean lighting = false;
     /** Whether enemy lighting is visible.
      * If lighting is enabled and this is false, a fog-of-war effect is partially achieved. */
+    public boolean dayNightCycle = false;
+    /** Whether Day & Night Cycle is enabled. */
+    public float dayNightCycleSpeed = 1.5f;
+    /** The light changing speed.*/
+    public boolean dayNightCycleDark = false;
+    /** Whether the light getting darker or brighter.*/
     public boolean enemyLights = true;
```

Comment:
```suggestion
    public boolean lighting = false;
    /** Whether enemy lighting is visible.
     * If lighting is enabled and this is false, a fog-of-war effect is partially achieved. */
    public boolean enemyLights = true;
    /** Whether Day & Night Cycle is enabled. */
    public boolean dayNightCycle = false;
    /** The light changing speed.*/
    public float dayNightCycleSpeed = 1.5f;
    /** Whether the light getting darker or brighter.*/
    public boolean dayNightCycleDark = false;
```

Do you even know how these comments work?